client/connectdlg_common.cpp

```diff
@@ -274,7 +222,7 @@ bool client_start_server(void)
     return FALSE;
   }
 
-  storage = freeciv_storage_dir();
+  storage = QString(freeciv_storage_dir());
```

Comment:
`QString::fromUtf8` for those users with an accent in their name (all Freeciv strings are UTF-8)